### ISSUE TITLE
Blocked Menu when using iOS Safari (also Firefox) mobile

### ISSUE THREAD

**Issue by kfe908** [snapappointments/bootstrap-select#2384]
The problem still exist, using v1.13.0 and Bootstrap 4.0.0

Using `$('.selectpicker').selectpicker('mobile') `does not work at all, the selectpicker dismisses the native picker instantly and is BLOCKING the entire select/dropdown. 

Using `$.fn.selectpicker.Constructor.DEFAULTS.mobile = true; ` will unblock the menu but will not use the native picker, the user will have a bad website experience instead using the regular desktop picker.

This problem has been discussed for 2 years. Is there any solution to that basic feature and  important bug?

**Comment by caseyjhol** [comment]
Duplicate of #1828. Please 👍 to vote and subscribe 🔉 for updates. This will be fixed in the next release.
